src/views/Swap/Swap.vue

```diff
@@ -688,6 +686,8 @@ export default {
       } else {
         min = liqualityMarket
           ? BN(liqualityMarket.min)
+          : !this.fiatRates[this.asset]
```

Comment:
I am not a big fan of these nested ternary operators (even if it is shorter by code perspective).
I will approve it, but we should consider refactoring this later